sys-cluster/pacemaker/files/pacemaker-2.1.2-command-not-found.patch

```diff
@@ -0,0 +1,11 @@
+--- pacemaker-Pacemaker-2.1.2/configure.ac
```

Comment:
change to a/, and b/, and link to the relevant bug(s) at the top of the patch

Reply:
Removed as it is not required